src/h3lib/lib/linkedGeo.c

```diff
@@ -100,6 +109,19 @@ LinkedGeoCoord* addLinkedCoord(LinkedGeoLoop* loop, const GeoCoord* vertex) {
     return coord;
 }
 
+/**
+ * Free all allocated memory for a linked geo loop. The caller is
+ * responsible for freeing memory allocated to input loop struct.
+ * @param loop Loop to free
+ */
+void destroyLinkedGeoLoop(LinkedGeoLoop* loop) {
+    for (LinkedGeoCoord *currentCoord = loop->first, *nextCoord;
```

Comment:
nit: Move `LinkedGeoCoord *nextCoord;` onto it's own line? I'm not sure if we have a policy of permitting/forbidding multiple declarations per line, but I would suggest against it.

Reply:
I can move it out. This is just a straight code move, though, the code hasn't changed from what it was.

Reply:
There's probably a clang-format setting for that. I'll try finding it.